.github/workflows/FissionPackage.yml

```
@@ -22,9 +22,25 @@ jobs:
       run: |
         echo "timestamp=$(date +'%Y-%m-%dT%H-%M-%S')" >> $GITHUB_OUTPUT
 
+    - name: Cache downloaded assets
+      id: cache-assets
+      uses: actions/cache@v2
+      with:
+        path: fission/public/
+        key: ${{ runner.os }}-assets-v1
+        restore-keys: |
+          ${{ runner.os }}-assets-
+
+    - name: Download assets if not cached
+      if: steps.cache-assets.outputs.cache-hit != 'true'
+      run: |
+        cd fission
+        curl -o public/assetpack.zip https://synthesis.autodesk.com/Downloadables/assetpack.zip && unzip -o public/assetpack.zip -d public/
```

I would rather the same tools be used when we download and unzip assets in workflows, even if the results are the same (tar vs unzip). This is the tiniest nitpick though.

```suggestion
        npm run assetpack
```

---

I actually removed the assetpack as that will be provided by the server.